web/frontend/src/components/remote-control-cards.svelte

```diff
@@ -109,6 +109,7 @@ const getStatus = (
     {#each filterSubtype($services, 'slam') as { name } (name)}
       <Slam
         {name}
+        motionResourceNames={filterSubtype($services, 'motion')}
```

Comment:
@ethanlook just want to confirm: Is this a safe change to make without coordinating or do we also need to update the APP's call sites for the slam component?

Reply:
This will require call site updates in app or the next RDK version will cause PR CI builds to fail, since this prop is non-optional.

Reply:
I'd imagine this is safe. At first I was thinking that this is the `<Slam2D>` prime component but since that API since that hasn't changed I don't imagine this causing issues in App.

Reply:
@micheal-parks could I trouble to please give this a review today? I want to get this in before the RC candidate is cut today as this issue affects users.